src/components/Auth/Auth/Login/IncompleteProfile/incompleteProfile.module.css

```diff
@@ -0,0 +1,130 @@
+@import '//netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.css';
```

Comment:
this is not a link la, remove this if youre not using it or move the import to main.module.css with the correct link if its needed